sqlalchemy_logger/logger.py

```diff
@@ -29,66 +32,71 @@ def check_owner(instance):
 
 
 def session_handler(session):
-    session_info = {}
-    session_instance = ""
-    method = ""
+    sessions_info = {}
```

Comment:
also think whether this kind of logging guarantees that log statements go in the same order as operations. That's fine if they don't, but you need to be aware of this.